css/style.css

```diff
@@ -0,0 +1,609 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* index.html - start*/
+
+.main-nav {
+    position: sticky;
+    top: 0.5rem;
+    z-index: 22222;
+    border: 3px solid #34dac9;
+    border-radius: 1rem;
+    margin: 0.5em;
+}
+
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.625rem;
+    padding: 0.2rem 0;
+    margin: .4rem;
+    position: sticky;
+    top: 0.5rem;
+}
+
+.navbar ul li {
+    margin: 0 0.5rem;
+    list-style: none;
+    display: inline-block;
+    position: relative;
+}
+
+.navbar ul li a {
+    color: rgb(0, 0, 0);
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.navbar ul li a:hover {
+    color: #34dac9;
+}
+
+.navbar ul li::after {
+    content: '';
+    height: 3px;
+    width: 0;
+    background: #34dac9;
+    left: 0;
+    bottom: -5px;
+    position: absolute;
+}
+
+.navbar ul li:hover::after {
+    width: 100%;
+    transition: 0.3s ease-out;
+}
+
+.poza {
+    padding: .5rem;
+    width: 97%;
+    margin: 0.5rem auto;
+    height: 75vh;
+    background-image: url(../img/poza.jpg) ;
+    background-size: cover;
+    background-position: center;
+}
+
+.s-container {
+    max-width: 100%;
+    margin: 0.5rem;
+}
+.p-container {
+    background-color: rgb(52, 218, 201, 0.3);
+    border-radius: 1rem;
+}
+
+.p-articol {   
+    padding: 1rem;
+    text-align: justify;
+    font-size: 1.2rem;
+    line-height: 1.5rem;
+    text-align: center;
+}
+
+.blog {
+    position: absolute;
+    top: 75%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: #34dac9;
+    color: rgb(0, 0, 0);
+    font-size: 1rem;
+    padding: 0.8rem 1.2rem;
+    border-radius: 0.4rem;
+    text-align: center;
+    text-decoration:none; 
+}
+
+.suplimente {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0.25rem 0;
+    gap: 0.6rem;
+}
+
+.poza-suplimente {
+    padding: 1rem;
+    width: 100%;
+    animation-name: slide;
```

Comment:
Din cauza animației, atunci când se încarcă pagina apare bară de scroll orizontală.

Reply:
La asta nu stiu ce sa ii fac sa nu apara, am vrut sa folosesc putin JS dar nu stiu ce nu am facut bine si nu mi-a mers codul, am vrut ca atunci cand dai scroll sa vina imaginea din stanga cu vine cu transform de la -100%, am incerc in codul de js cu console.log sa vad daca merge scrolul si mergea dar nu se activa cand dadeam scrol sa ajunga le element la imagine .. . Acesta a fost codul de JS 

// document.addEventListener("scroll", function() {
//   const element = document.querySelector(".poza-suplimente");
//   const rect = element.getBoundingClientRect();

//   if (rect.top < window.innerHeight) {
//     element.style.animation = "slide"; 
//   }
// });